src/scss/settings/_variables.scss

```diff
@@ -128,6 +128,8 @@ $img-path-url              : '/assets/img' !default; // Usage: background: url('
 $icon-path                 : '#{$img-path}/icons' !default;
 $font-path                 : 'dist/fonts' !default;
 
+$star-icon-path--filled    : '' !default;
```

Comment:
Don't know if you'd want/need to but you could set a default path here? e.g. `#{$icon-path}/stars/star--filled.svg` or similar?

Reply:
Yeah that's a good shout